.ci_support/linux_python2.7.yaml

```diff
@@ -9,7 +9,7 @@ channel_targets:
 docker_image:
 - condaforge/linux-anvil-comp7
 hdf5:
-- 1.10.5
+- 4.6.3
```

Comment:
Something is going with re-render? This version does not exist!

@conda-forge/core

Reply:
That's wild! https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L412 it seems the pinning file is correct.

Reply:
Yeah. I cannot even reproduce that. Not sure what happened!